### ISSUE TITLE
Bug: Latest Update breaks the better command palette

### ISSUE THREAD

**Issue by chrisgrieser** [AlexBieg/obsidian-better-command-palette#25]
I don't now what, but the "open better command palette" command somehow stopped working?

Also, I tried reinstalling the plugin disabling all other plugins. Still not working. :(

```log
app.js:1 Command failed to execute:  obsidian-better-command-palette:open-better-commmand-palette
app.js:1 TypeError: Cannot read property 'name' of undefined
    at eval (eval at <anonymous> (app.js:1), <anonymous>:15:13972)
    at Array.map (<anonymous>)
    at g.initialize (eval at <anonymous> (app.js:1), <anonymous>:15:13924)
    at P.updateActionType (eval at <anonymous> (app.js:1), <anonymous>:15:18058)
    at new P (eval at <anonymous> (app.js:1), <anonymous>:15:17116)
    at Object.callback (eval at <anonymous> (app.js:1), <anonymous>:15:23968)
    at SR (app.js:1)
    at t.executeCommandById (app.js:1)
    at t.eval (eval at <anonymous> (app.js:1), <anonymous>:74:11668)
    at t.o [as executeCommandById] (eval at <anonymous> (app.js:1), <anonymous>:23:77225)
t.executeCommandById @ app.js:1
eval @ VM154:74
o @ VM154:23
t.onTrigger @ app.js:1
t.handleKey @ app.js:1
t.onKeyEvent @ app.js:1
```

**Comment by AlexBieg** [comment]
Looking at it now

**Comment by AlexBieg** [comment]
Ah I think I found the issue. I think you have a command pinned that no longer exists. Probably due to a removed plugin. I should have a fix out in a few minutes. In the meantime you should be able to fix it, but adding back whatever plugin that was and un-pinning the command.

**Comment by chrisgrieser** [comment]
ah, got it. can confirm, re-installing the plugin makes BCP work again.

**Comment by AlexBieg** [comment]
@chrisgrieser Just released 0.7.1. That should fix the issue. FYI sometimes obsidian takes a minute to realize that a plugin has been updated and lets you update it.

**Comment by chrisgrieser** [comment]
thanks for the quick fix :)
